### ISSUE TITLE
Missing a function, This fixes FORE000005 and NEWS 000005

### ISSUE THREAD

**Issue by weirdal3333** [RiiConnect24/.VFF-File-Downloader-for-Dolphin#20]
![unknown](https://user-images.githubusercontent.com/3537180/171965643-e18816df-a5f8-40da-af6d-584ac6dc9298.png)

**Comment by KcrPL** [comment]
Could you explain?

**Comment by weirdal3333** [comment]
When getting the errors in the title, I had done all of the normal troubleshooting steps and had run your downloader program, which did download the files correctly... Still an error though. 

I decided to check the wiiconnect24 config file, as this is dolphin and checking the config files is easy. I changed all the "wii.com" references to "rc24.xyz". Upon restarting the dolphin instance, I noticed that the channels had actual news on them, and upon turning the standby connection on (it always defaults to off for some reason, the channels worked for the first time in the several days of troubleshooting!

I figured that I should mention the fix here because I see a lot of people with the error in that discord, and it doesn't always get solved with the tool. A simple check on the "nwc24msg.config" file for "wii.com" could be helpful in diagnosing issues at least.
